contracts/pair_xyk_sale_tax/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "astroport-pair-xyk-sale-tax"
+version = "1.4.0"
```

Comment:
I think this should be bumped to 1.4.1

Reply:
Or set back to 1.0.0 as it's a separate contract from the standard xyk? Doesn't matter either way to me, I can change it to 1.4.1 if you prefer.